function_lab.js

```diff
@@ -0,0 +1,297 @@
+// # Function Exercises
```

Comment:
Good job, I see your developing your skills in functions and scopes. Reach out to me if you would like to complete this assignment with my guidance . I'm here for you :)